### DISCUSSION TITLE
Allow field key names to go multiline if text width > page width

### DISCUSSION THREAD

**Discussion by lucasshipsides** [directus/directus#13136] [Feature Requests]
Hi folks

Came across this when someone was using a form on their mobile. The text for the field key name (which unfortunately needs to be that long) only displays as a single line and doesn't wrap if the page width is smaller than the text width.

It looks like dividers and notes are properly responsive and do correctly wrap when the page is narrow, so hopefully this is is an easy fix for the headers!

Screenshot below should be obvious, the header text goes beyond the word "undergraduate":

![Screenshot_20220505-160420_Chrome](https://user-images.githubusercontent.com/104183093/166917521-2996f80a-a3b3-4db0-aca0-d6697b82f5bd.jpg)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
